### PR TITLE
Fix cmake install

### PR DESCRIPTION
**Pull request by Waqar144**
It now works and produces the same output as qmake. Also you don't need to run lrelease manually anymore with cmake.

### CODE REVIEW

**Comment by Waqar144** [comment]
One thing that is left a la cmake is that it still compiles the nssspellchecker on macos. That was removed long ago and we use hunspell everywhere IIRC so cmake should do the same.

**Comment by Waqar144** [comment]
> One thing that is left a la cmake is that it still compiles the nssspellchecker on macos. That was removed long ago and we use hunspell everywhere IIRC so cmake should do the same.

Done

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -473,16 +474,6 @@ list(APPEND SOURCE_FILES
 )
 endif()
```
Comment (pbek) [inline src/CMakeLists.txt:475]: This line needs to be remove too then, right?

**Comment by pbek** [comment]
Thank you!

**Pull request merged**